ml-conversational-analytic-tool/tests/test_commentAnalysis.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+from unittest.mock import MagicMock
```

Comment:
The best practice is to import only modules, not classes (ie MagicMock).

Reply:
Done